src/PhpParser/Visitor/MutationsCollectorVisitor.php

```diff
@@ -46,9 +46,9 @@
 final class MutationsCollectorVisitor extends NodeVisitorAbstract
 {
     /**
-     * @var Mutation[]
+     * @var iterable[]
      */
-    private $mutations = [];
+    private $iterators = [];
```

Comment:
```suggestion
    private $mutationChunks = [];
```

? (terminology taken after `array_chunk()`) - requires to refactor the occurrences below as well including the variables in the `foreach` statements

Reply:
They're chunks indeed.